ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -230,6 +231,15 @@ export const ConceptHomepage = fp.flow(withCurrentCohortSearchContext(), withCur
       this.setState({conceptDomainList, conceptSurveysList});
     }
 
+    // Temp function to use the correct endpoint based on the enableStandardSourceDomains config flag
+    getDomainCounts(domain: string) {
+      const {id, namespace} = this.props.workspace;
+      const {currentInputString} = this.state;
+      return serverConfigStore.get().config.enableStandardSourceDomains
+        ? cohortBuilderApi().findDomainCountByStandardSource(namespace, id, domain, true, currentInputString)
```

Comment:
Just passing `true` as standard param for now. Will update when we add the advanced search option to the searchbar.

Reply:
Thanks. Per our conversation offline, the standard param will remain `true` since there will be no advanced search option on the concept homepage.